mk/extbld/arch-embox-clang

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
I slightly changed the script. Maybe we can merge this back to `arch-embox-gcc` if we pass some env variable about clang and check for it. 

````
--- mk/extbld/arch-embox-gcc    2016-10-06 12:41:09.016126951 +0300
+++ mk/extbld/arch-embox-clang  2017-01-12 13:11:02.483323783 +0300
@@ -9,7 +9,7 @@
 
 cmd=$(basename $0)
 case $cmd in
-       *-gcc) C_CXX_FLAGS="$EMBOX_IMPORTED_CPPFLAGS $EMBOX_IMPORTED_CFLAGS";;
+       *-clang|*-gcc) C_CXX_FLAGS="$EMBOX_IMPORTED_CPPFLAGS $EMBOX_IMPORTED_CFLAGS";;
        *-g++) C_CXX_FLAGS="$EMBOX_IMPORTED_CPPFLAGS $EMBOX_IMPORTED_CXXFLAGS";;
        *)     echo "Unknown flags for $cmd"; exit 1;;
 esac
@@ -33,6 +33,6 @@
 
 ARG_LINE="$ARG_LINE $EMBOX_IMPORTED_CPPFLAGS"
 PWD_ARG_LINE="$(for i in $ARG_LINE; do echo ${i/$PWD/.}; done)"
-# echo "$EMBOX_CROSS_COMPILE${cmd#arch-embox-} $@ $PWD_ARG_LINE" >&2
-$EMBOX_CROSS_COMPILE${cmd#arch-embox-} "$@" $PWD_ARG_LINE
+# we hardcore target because we should rely on installed GCC toolchain
+clang "$@" -target arm-none-eabi $PWD_ARG_LINE
 exit $?
````